clang/test/Sema/warn-lifetime-analysis-nocfg.cpp

```diff
@@ -499,3 +518,28 @@ std::string_view test2(int i, std::optional<std::string_view> a) {
   return std::move(a.value());
 }
 }
+
+namespace GH100526 {
+void test() {
+  std::vector<std::string_view> t1 = {std::string()}; // expected-warning {{object backing the pointer will be destroyed at the end}}
```

Comment:
Can you try adding more than one args to init list for `std::vector`.
Eg: `std::vector<std::string_view> s = {std::string_view(),  std::string()}`

Does it work atm ?

Reply:
Yeah, it works. Added this case.

Reply:
Great. Thanks.